presto-orc/src/main/java/io/prestosql/orc/reader/BooleanStreamReader.java

```diff
@@ -103,10 +103,16 @@ public Block readBlock(Type type)
             }
             dataStream.getSetBits(type, nextBatchSize, builder);
         }
+        else if (dataStream == null) {
+            presentStream.skip(nextBatchSize);
```

Comment:
I wonder if we shouldn't validate that all `presentStream` bits are `false` (to prevent reading corrupted files),
@dain ?

Reply:
I don't think it is necessary.  There are plenty of other places in this code base were we don't validate everything.